firebase-firestore/src/test/java/com/google/firebase/firestore/model/ObjectValueBuilderTest.java

```diff
@@ -31,9 +31,9 @@
 @Config(manifest = Config.NONE)
 public class ObjectValueBuilderTest {
   private String fooString = "foo";
-  private Value fooValue = valueOf(fooString);
+  private Value fooValue = TestUtil.wrap(fooString);
```

Comment:
Static import `TestUtil.wrap` (to match other usages above).

Reply:
Done